task/tkn-bundle/0.1/tkn-bundle.yaml

```diff
@@ -32,17 +36,17 @@ spec:
     env:
     - name: HOME
       value: "$(params.HOME)"
+    - name: TASK_FILE
+      value: tekton_task_files
   steps:
-  - image: quay.io/openshift-pipeline/openshift-pipelines-cli-tkn:5.0
-    name: build
+  - image: quay.io/redhat-appstudio/appstudio-utils:3ef7e414efcb0e625d72ed4b025ea22ab362c60d
```

Comment:
I don't think we should use the `appstudio-utils` image. IIUC its purpose is mostly to assist in the CI of this repo. It is not meant to be used by folks building images in Konflux.

Presumably, you grabbed this image because it's a convenient way to have an image with yq installed. I don't have a good alternative other than maybe just re-use the `quay.io/redhat-appstudio/hacbs-test` image used in the [fbc-related-image-check](https://github.com/redhat-appstudio/build-definitions/blob/3ef7e414efcb0e625d72ed4b025ea22ab362c60d/task/fbc-related-image-check/0.1/fbc-related-image-check.yaml#L20) task. But maybe check with the integration folks that this is ok.

Reply:
I was unsure on what to use. Thanks for the suggestion, I'll check with the test team.

Reply:
actually, shellspec times out when I try and use that image. Not exactly sure why.

Reply:
I agree that the bundle operations should use that image and I wanted to add the yaml modifications to the existing step, but using sed and awk was messy and looked error prone, so I added the second step for the file operations.

Reply:
Found an alternative image that should be OK. registry.access.redhat.com/ubi9/toolbox@sha256:7391628396216c011ed3a310f1fa54c6a9221e36f7fa59c94ae7796de51e7a25